x-pack/test/security_solution_endpoint_api_int/apis/resolver/common.ts

```diff
@@ -44,6 +380,7 @@ export const createAncestryArray = (events: Event[]) => {
 /**
  * Check that the given lifecycle is in the resolver tree's corresponding map
  *
+ * @deprecated use verifyTree
```

Comment:
These deprecated functions can be removed once the front end is switched to use the new `/tree` api